### PR TITLE
feat: Add basic deterministic data with configurable avg updates freq

### PR DESCRIPTION
**Pull request by vetleaa**
**Summary**

- Add very simple deterministic data generation with configurable update frequency
_(Should produce the same values over time as long as the seed remains the same)_
- Restructure backend a bit _(I like `src`, okay?)_
- Equal width number characters

### CODE REVIEW

File: backend/go.mod [hunk 1]
```diff
@@ -2,4 +2,4 @@ module github.com/implocell/solid-stream
 
 go 1.19
 
-require github.com/gorilla/websocket v1.5.0 // indirect
+require github.com/gorilla/websocket v1.5.0
```
Comment (vetleaa) [inline backend/go.mod:5]: 🤷

File: backend/src/main.go [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import (
+	"encoding/json"
+	"log"
+	"math/rand"
+	"net/http"
+	"time"
+
+	"github.com/implocell/solid-stream/src/ticker"
+
+	"github.com/gorilla/websocket"
```
Comment (vetleaa) [inline backend/src/main.go:12]: Apparently this is no longer maintained :o

File: backend/go.mod (hunk 1 above)
Comment (Implocell) [reply backend/go.mod:5]: Seems correct, should have runned go mod tidy on commit

File: backend/src/main.go (hunk 2 above)
Comment (Implocell) [reply backend/src/main.go:12]: Nope, but from what I've read it's the most mature one for now, even if it is no longer maintained.

**Review by Implocell — APPROVED** [review]

File: backend/src/main.go (hunk 2 above)
Comment (vetleaa) [reply backend/src/main.go:12]: Yup, I got the same impression

It's just a bit sad that the most reliable solution for websockets in go is one that isn't maintained anymore :(
Though I suspect one of the alternatives will get more attention now that this one is on ice

**Pull request merged**